cli/sql_util.go

```diff
@@ -173,8 +174,11 @@ func formatVal(val interface{}) string {
 	case nil:
 		return "NULL"
 	case []byte:
-		// Ensure that binary protobufs print escaped.
-		return fmt.Sprintf("%q", t)
+		if !utf8.Valid(t) {
+			// Ensure that binary protobufs print escaped.
```

Comment:
I think you'll want to update this comment; there's no guarantee that a protobuf will print escaped, right?